test/e2e/node/container_probe.go

```diff
@@ -14,17 +14,15 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package e2e
+package node
```

Comment:
So this is the package for shared tests? I feel like it should be named something like `common` or `shared`. IMHO ideally the shared tests would be in `test/e2e`, with the node-specific tests in `test/e2e/node` and the cluster specific tests in `test/e2e/cluster`. But that's a big change :)